collections/hacking-minecraft/index.md

```diff
@@ -1,34 +1,43 @@
 ---
 items:
  - docker/dockercraft
+ - Bukkit/Bukkit
+ - SpigotMC/BungeeCord
+ - PaperMC/Waterfall
+ - PaperMC/Paper
+ - PaperMC/Velocity
+ - PurpurMC/Purpur
+ - MinecraftForge/MinecraftForge
+ - FabricMC/fabric
+ - Luohuayu/CatServer
  - minefold/hubot-minecraft
  - leereilly/hubot-minecraft-skin
  - overviewer/Minecraft-Overviewer
  - cuberite/cuberite
  - pmmp/PocketMine-MP
  - msmhq/msm
+ - PojavLauncherTeam/PojavLauncher
  - essentials/Essentials
  - VazkiiMods/Botania
  - MightyPirates/OpenComputers
  - PrismarineJS/mineflayer
  - EngineHub/WorldEdit
- - SpigotMC/BungeeCord
  - walterhiggins/ScriptCraft
- - MinecraftForge/MinecraftForge
  - ddevault/TrueCraft
  - MachineMuse/MachineMusePowersuits
+ - MCSManager/MCSManager
+ - Fallen-Breath/MCDReforged
  - micdoodle8/Galacticraft
- - Bukkit/Bukkit
  - GlowstoneMC/Glowstone
  - MovingBlocks/Terasology
  - Zerite/CraftLib
- - PaperMC/Paper
  - CaffeineMC/sodium-fabric
- - FabricMC/fabric
- - lambda-client/lambda/
+ - lambda-client/lambda
  - nerdsinspace/nocom-explanation
+ - CCBlueX/LiquidBounce
 display_name: Hacking Minecraft
-created_by: leereilly
+created_by: leereilly, YaoSiQian
```

Comment:
You're adding to this but you didn't create it.

```suggestion
created_by: leereilly
```

Reply:
Hello hacker